yjit/src/codegen.rs

```diff
@@ -1435,6 +1435,32 @@ fn gen_splatarray(
     Some(KeepCompiling)
 }
 
+// call to_hash on hash to keyword splat before converting block
```

Comment:
Can you include an example in the comment of a call that would generate this?

Reply:
sure https://github.com/ruby/ruby/pull/9711/commits/6a43c3efff0a0284c8b8bc7b8fbc79026020cc98